src/ochubryk/Hometask2/price.py

```diff
@@ -0,0 +1,36 @@
+# Напишите программу, которая считает общую цену.
+M_rubles = int(input('Введите рубли: '))
+N_pennies = int(input('Введите копейки: '))
+L_count = int(input('Введите количество товара: '))
```

Comment:
it's not python namings. in Python variables are named in  snake case

Reply:
Fixed